ocp_install_env.sh

```diff
@@ -312,8 +315,15 @@ spec:
   bootMACAddress: $mac
   bmc:
     address: $address
-    credentialsName: ${name}-bmc-secret
-EOF
+    credentialsName: ${name}-bmc-secret"
 
+        echo "${secret}${bmh}" >> "${outdir}/${host_output}"
+
+        echo "${secret}" >> "${outdir}/extras/extraworker-${worker_index}-secret"
+        echo "${bmh}" >> "${outdir}/extras/extraworker-${worker_index}-bmh"
+        ((worker_index+=1))
     done
-}
+
+    # This secret will be used by the e2e test to create/destroy additional workers when required
+    oc create secret generic extraworkers-secret --from-file="${outdir}/extras/" --dry-run=client --output=yaml > "${outdir}/extraworkers-secret.yaml"
```

Comment:
This can't run before the cluster is up
```
+(ocp_install_env.sh:328): generate_ocp_host_manifest(): oc create secret generic extraworkers-secret --from-file=ocp/ostest/extras/ --dry-run=client --output=yaml                                               
W1125 12:35:05.692447 1833488 loader.go:221] Config not found: /home/derekh/dev-scripts/ocp/ostest/auth/kubeconfig                 
```

Reply:
Ok thanks, was under the impression that `--dry-run=client` was enough, will fix it